articles/architecture-scenarios/b2c/introduction.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Introduction to Consumer Identity and Access Management Implementation
+description: High-level overview of the Consumer Identity and Access Management Implementation guide
+toc: false
+topics:
+    - b2c
+    - ciam
+contentType: concept
+useCase:
+  - implementation
+---
+# The Consumer Identity and Access Management Implementation Guide: An Introduction
+
+This guide focuses on showing you how to integrate Auth0 within a Consumer Identity and Access Management (CIAM)-style project. Our recommendations are based on real-world customer implementation experience, and we hope that the advice we provide helps you set your project up for success.
```

Comment:
When putting the original B2C Welcome together, the consensus was that the language used should be as prescriptive and definitive as possible. So perhaps something like this would work better: 
```suggestion
This guide focuses on showing you how to integrate Auth0 within a Consumer Identity and Access Management (CIAM)-style project. Our recommendations are based on real-world customer implementation experience, and by following the guidance provided you will set your project up for success.
```